DomainContent/Korea/Brush/Marker_Server.js

```diff
@@ -0,0 +1,191 @@
+// Marker_Server.js
+//
+// Copyright 2018 High Fidelity, Inc.
+// Created by Milad Nazeri and Robin Wilson 7/5/2018
+// Expands on fingerPaint.js created by David Rowe 15 Feb 2017
+// 
+// Server script to the marker. Creates lines server side to allow users without rez rights to draw polylines.
+// Listens to Marker_Client for marker positions. Works with Marker_Client.js.
+//
+// Distributed under the Apache License, Version 2.0.
+// See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+(function () {
+    var STROKE_COLOR = {
+            red: 250,
+            green: 0,
+            blue: 0
+        },
+        ERASE_SEARCH_RADIUS = 0.1, // m
+        isDrawingLine = false,
+        entityID,
+        basePosition,
+        strokePoints,
+        strokeNormals,
+        strokeWidths,
+        timeOfLastPoint,
+        MIN_STROKE_LENGTH = 0.005, // m
+        MIN_STROKE_INTERVAL = 66, // ms
+        MAX_POINTS_PER_LINE = 70; // Hard-coded limit in PolyLineEntityItem.h.
+
+    var currentAvatarOrientation;
+
+    function strokeNormal() {
+        return Vec3.multiplyQbyV(currentAvatarOrientation, Vec3.UNIT_NEG_Z);
+    }
+
+    function MarkerServer() {
+
+    }
+
+    MarkerServer.prototype = {
+        remotelyCallable: [
+            "startLine",
+            "drawLine",
+            "finishLine",
+            "cancelLine",
+            "eraseClosestLine",
+            "tearDown"
+        ],
+        preload: function (id) {
+
+        },
+        startLine: function (id, params) {
+            console.log("startLine");
```

Comment:
Wrap these in debug statements please!

I would also suggest that we might want to confirm that the params array has the indices we expect so we can fail gracefully if this is called with incomplete data instead of hitting an out of bounds error when trying to access the elements